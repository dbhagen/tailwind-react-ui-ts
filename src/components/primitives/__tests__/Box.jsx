import React from 'react'
import { shallow } from 'enzyme'

import { Box } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign({}, testProps)

  const wrapper = shallow(<Box {...props}>Hello World</Box>)

  return {
    props,
    wrapper,
  }
}

describe('Box', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
