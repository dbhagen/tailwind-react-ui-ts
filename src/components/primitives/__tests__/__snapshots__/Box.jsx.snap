// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders matching snapshot 1`] = `
<WithTheme(Base)
  inline={false}
  inlineBlock={false}
  is="div"
>
  Hello World
</WithTheme(Base)>
`;
